h2o-r/h2o-package/R/adaboost.R

```diff
@@ -0,0 +1,169 @@
+# This file is auto-generated by h2o-3/h2o-bindings/bin/gen_R.py
+# Copyright 2016 H2O.ai;  Apache License Version 2.0 (see LICENSE for details) 
+#'
+# -------------------------- AdaBoost Model in H2O -------------------------- #
+#'
+#' Build an AdaBoost model
+#' 
+#' Builds an AdaBoost model on an H2OFrame.
+#'
+#' @param x (Optional) A vector containing the names or indices of the predictor variables to use in building the model.
+#'        If x is missing, then all columns except y are used.
+#' @param y The name or column index of the response variable in the data. 
+#'        The response must be either a numeric or a categorical/factor variable. 
+#'        If the response is numeric, then a regression model will be trained, otherwise it will train a classification model.
+#' @param training_frame Id of the training data frame.
+#' @param model_id Destination id for this model; auto-generated if not specified.
+#' @param ignore_const_cols \code{Logical}. Ignore constant columns. Defaults to TRUE.
+#' @param categorical_encoding Encoding scheme for categorical features Must be one of: "AUTO", "Enum", "OneHotInternal", "OneHotExplicit",
+#'        "Binary", "Eigen", "LabelEncoder", "SortByResponse", "EnumLimited". Defaults to AUTO.
+#' @param weights_column Column with observation weights. Giving some observation a weight of zero is equivalent to excluding it from
+#'        the dataset; giving an observation a relative weight of 2 is equivalent to repeating that row twice. Negative
+#'        weights are not allowed. Note: Weights are per-row observation weights and do not increase the size of the
+#'        data frame. This is typically the number of times a row is repeated, but non-integer values are supported as
+#'        well. During training, rows with higher weights matter more, due to the larger loss function pre-factor. If
+#'        you set weight = 0 for a row, the returned prediction frame at that row is zero and this is incorrect. To get
+#'        an accurate prediction, remove all rows with weight == 0.
+#' @param nlearners Number of AdaBoost weak learners. Defaults to 50.
+#' @param weak_learner Choose a weak learner type. Defaults to DRF. Must be one of: "AUTO", "DRF", "GLM", "GBM". Defaults to AUTO.
```

Comment:
```suggestion
#' @param weak_learner Choose a weak learner type. Must be one of: "AUTO", "DRF", "GLM", "GBM". Defaults to AUTO, which means DRF.
```

Reply:
Thank, I will do it as part of large commit though since it must be updated in AdaBoostV3 and rebuild.